### PR TITLE
GHC 8.2.

### PR DESCRIPTION
**Pull request by krisajenkins**
All the necessary changes to make servant-ekg with with GHC 8.2.

You /might/ want to hold off on this until 8.2 goes official on stackage. (You might want to ignore this PR entirely. You are free! :-) Nevertheless, here is the code. :-) )

### CODE REVIEW

**Comment by jkachmar** [comment]
@alpmestan Not sure who the correct maintainer to ping on this would be, but I saw that you merged the other PRs here. Anything blocking this?

**Comment by alpmestan** [comment]
Oh, sorry that this one was forgotten about. I suppose we'll want to change the bound on `servant` but in principle this patch looks fine to me. How about supporting 8.4 while we're at it? =)

@phadej thoughts?

**Comment by jkachmar** [comment]
I haven't tested anything on 8.4 yet, but (as far as I can tell) this just needs a very minor fix to be compliant with `servant-0.12` as well. (Just defining a few instances to work with `Description` and `Summary`; I have it working on a local fork.)

File: servant-ekg.cabal [hunk 1]
```diff
@@ -17,9 +17,9 @@ source-repository HEAD
 library
   exposed-modules:     Servant.Ekg
   hs-source-dirs:      lib
-  build-depends:       base >=4.7 && < 4.10
+  build-depends:       base == 4.*
```
Comment (phadej) [inline servant-ekg.cabal:20]: :( `>= 4.7 && <4.11`

Don't relax lower bound, it's there for a reason.

**Review by phadej — APPROVED** [review]
This is fine (one small comment). I'll merge and adopt for a 0.13 release

**Comment by jkachmar** [comment]
@phadej Any movement on this now that 0.13 is out?

**Comment by phadej** [comment]
Thanks for reminder, `servant-ekg` isn't in my `servant-universe` so I completely forgot about it. Will make release this week.

**Comment by jessekempf** [comment]
@phadej: Can you make a release this year?

**Comment by phadej** [comment]
In fact, I cannot. @jkarni could you give @alpmestan and me maintainer rights on Hackage?

**Comment by phadej** [comment]
Superseded by #7.

**Pull request closed without merging**